charts/flux2/values.yaml

```diff
@@ -2,6 +2,10 @@
 
 installCRDs: true
 
+# -- Implement the patches for Multi-tenancy lockdown. 
+# See https://fluxcd.io/docs/installation/#multi-tenancy-lockdown
+multiTenancyLockdown: false
```

Comment:
Can we please allow users to change the default service account. Something like this:

```yaml
multitenancy:
  enabled: false
  defaultServiceAccount: "default"
```

Reply:
Done